lib/gingr/config.rb

```diff
@@ -27,6 +27,10 @@ class << self
       attr_accessor :geofile_ingestion_dirname, :reference_urls, :default_options
 
       include Config
+
+      def getopt(optname)
```

Comment:
This method turns out to be really useful for all the cases we need a fallback value. I thought it made sense for it to live in the Config class since it's general purpose.